style.json

```diff
@@ -4609,6 +4863,10 @@
           ">=",
           "rank",
           3
+        ],
+        [
+          "has",
+          "iso_a2"
```

Comment:
What's the use of this filter ?

Reply:
It allows better styling for disputed country (they should be present, but with different styling than actual country). These countries does not have a iso_a2 field.
See South Ossetia (partially recognised state) vs Georgia (state)